.github/workflows/ci.yml

```diff
@@ -42,31 +42,31 @@ jobs:
   integration:
     runs-on: "ubuntu-20.04"
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
I believe this was set to `true` to minimize GHA minutes.

Reply:
Not worried about GH minutes on a public repo, no charge on them. This will allow us to see the results of all the tests to help reduce dev time.

Reply:
Makes sense.